src/NServiceBus.AcceptanceTests/Satellites/When_ripping_out.cs

```diff
@@ -0,0 +1,174 @@
+namespace NServiceBus.AcceptanceTests.Satellites
+{
+    using System;
+    using System.Threading.Tasks;
+    using AcceptanceTesting;
+    using AcceptanceTesting.Customization;
+    using EndpointTemplates;
+    using Features;
+    using NServiceBus.Pipeline;
+    using NServiceBus.Routing;
+    using NUnit.Framework;
+    using Transport;
+
+    public class When_ripping_out : NServiceBusAcceptanceTest
+    {
+        [Test]
+        public async Task Should_receive_the_message()
+        {
+            var context = await Scenario.Define<Context>()
+                .WithEndpoint<Endpoint>(b => b.When((session, c) => session.SendLocal(new MyMessage())))
+                .WithEndpoint<Receiver>()
+                .Done(c => c.MessageReceived && c.OtherMessageReceived)
+                .Run().ConfigureAwait(false);
+
+            Assert.True(context.MessageReceived);
+        }
+
+        class Context : ScenarioContext
+        {
+            public bool MessageReceived { get; set; }
+            public bool OtherMessageReceived { get; set; }
+        }
+
+        class Endpoint : EndpointConfigurationBuilder
+        {
+            public Endpoint()
+            {
+                EndpointSetup<DefaultServer>(c =>
+                {
+                    var transport = c.ConfigureTransport();
+                    transport.Routing().RouteToEndpoint(typeof(MyOtherMessage), Conventions.EndpointNamingConvention(typeof(Receiver)));
+                    
+                    c.Pipeline.Replace("TransportReceiveToPhysicalMessageProcessingConnector", new BastardizingTheCoreForkConnector(OnMessage, OnError));
+                    
+                    c.DisableFeature<AutoSubscribe>();
+                    c.DisableFeature<Sagas>();
```

Comment:
that would definitely be stuff we could avoid by making the assembly scanning more configurable